code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -0,0 +1,376 @@
+#define IC_COMPONENTS_BASE		25
+#define IC_COMPLEXITY_BASE		75
+
+/obj/item/device/electronic_assembly
+	name = "electronic assembly"
+	desc = "It's a case, for building small electronics with."
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/electronic_assemblies.dmi'
+	icon_state = "setup_small"
+	flags_1 = NOBLUDGEON_1
+	var/max_components = IC_COMPONENTS_BASE
+	var/max_complexity = IC_COMPLEXITY_BASE
+	var/opened = FALSE
+	var/obj/item/stock_parts/cell/battery // Internal cell which most circuits need to work.
+	var/cell_type = /obj/item/stock_parts/cell
+	var/can_charge = TRUE //Can it be charged in a recharger?
+	var/charge_sections = 4
+	var/charge_tick = FALSE
+	var/charge_delay = 4
+	var/use_cyborg_cell = TRUE
+
+/obj/item/device/electronic_assembly/medium
+	name = "electronic mechanism"
+	icon_state = "setup_medium"
+	desc = "It's a case, for building medium-sized electronics with."
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Er... the medium and small ones are the same size, or is the large one meant to be not normal sized?